### DISCUSSION TITLE
Any plans on a GTK4 fork?

### DISCUSSION THREAD

**Discussion by PlatinumLucario** [GtkSharp/GtkSharp#338] [General]
I'm really curious if there will be any plans in the future for making a GTK4 fork of GtkSharp? I'm aware that GTK4 is quite new, which has got me thinking that I may someday try experimenting around with GTK4 code in C# once I'm more experienced with developing in GTK.

It's a very interesting thought though.

**Comment by cameronwhite** [comment]
See #166
There is https://github.com/gircore/gir.core which is a new binding generator that will support GTK4 and other GObject-based libraries
